src/test/java/core/basesyntax/service/TransactionProcessorTest.java

```diff
@@ -0,0 +1,89 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.database.Storage;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.impl.TransactionProcessorImpl;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.impl.BalanceHandler;
+import core.basesyntax.strategy.impl.OperationStrategyImpl;
+import core.basesyntax.strategy.impl.PurchaseHandler;
+import core.basesyntax.strategy.impl.ReturnHandler;
+import core.basesyntax.strategy.impl.SupplyHandler;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class TransactionProcessorTest {
+    private static List<Transaction> transactions;
+    private static TransactionProcessor processor;
+    private static OperationStrategy strategy;
+
+    @BeforeAll
+    static void beforeAll() {
+        Map<Operation, OperationHandler> operationMap = new HashMap<>();
+        operationMap.put(Operation.BALANCE, new BalanceHandler());
+        operationMap.put(Operation.SUPPLY, new SupplyHandler());
+        operationMap.put(Operation.PURCHASE, new PurchaseHandler());
+        operationMap.put(Operation.RETURN, new ReturnHandler());
+        strategy = new OperationStrategyImpl(operationMap);
+    }
+
+    @BeforeEach
+    void beforeEach() {
+        Storage.STORAGE.clear();
+        processor = new TransactionProcessorImpl(strategy);

```

Comment:
can you move processor initialization to BeforeAll and clear storage to AfterEach?